examples/dataset/search_embeddings/search_embeddings/upload_embeddings.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import os
+from typing import Iterator
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+import boto3
+import numpy as np
+import pandas as pd
+from kembed._clipper import StudioModel
+from kembed.util import extract_embeddings
+from kembed.util import load_embedding_model
+from PIL import Image
+from tqdm import tqdm
+
+import kolena
+from kolena._experimental.search import upload_dataset_embeddings
+from kolena.dataset import download_dataset
+
+
+BUCKET = "kolena-public-examples"
+DATASET = "coco-stuff-10k"
+IMAGE_S3_DIR = f"s3://{BUCKET}/{DATASET}/data/images/"
+LOCATOR_FIELD = "locator"
```

Comment:
Should be emphasis that image link are always stored in locator column? or is this a well known convention

Reply:
I feel like we need to beef up documentations in general, especially on these magic column names as well as how the ID columns are behaving. Could fall into this ticket: https://linear.app/kolena/issue/KOL-4992/documentation-how-to-format-your-dataset